AntWars/Board/Ants/Ant.cs

```diff
@@ -75,10 +75,9 @@ internal Ant(Board board, Player owner, int viewRange, int maxInventory, int mov
         /// Verringert die UnitsGone einer Ameise um einen bestimmten Prozentsatz.
         /// </summary>
         /// <returns>True bei Regenerierung, false wenn Ameise nicht in der Base steht oder bereits eine Aktion ausgeführt hat.</returns>
-        public bool restore() {
+        public bool recover() {
             if (!TookAction && isInBase()) {
-                // 0.2 kann später durch den bestimmten oder aufgewerteten Prozentsatz der Base ersetzt werden.
-                UnitsGone = Convert.ToInt32(Math.Ceiling(UnitsGone * (1 - 0.2)));
+                UnitsGone = Convert.ToInt32(Math.Ceiling(UnitsGone * (1d - (getBase().Recover / 10))));
```

Comment:
> getBase().Recover / 10

Integer kann man nicht ordentlich teilen, also entweder konvertieren oder in der Base direkt als Datentyp double/float deklarieren.
Hier würde bei Recover 1-9 immer 0 rauskommen.